test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SWAEndToEndTests.cs

```diff
@@ -0,0 +1,151 @@
+﻿using Microsoft.Azure.WebJobs.Script.WebHost;
+using Microsoft.Azure.WebJobs.Script.WebHost.Management;
+using Microsoft.Azure.WebJobs.Script.Workers.Rpc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Integration.WebHostEndToEnd
+{
+    /// <summary>
+    /// Tests verifying the Static Web Apps deployment configuration.
+    /// </summary>
+    public class SWAEndToEndTests : IClassFixture<SWAEndToEndTests.TestFixture>
+    {
+        private TestFixture _fixture;
+
+        public SWAEndToEndTests(TestFixture fixture)
```

Comment:
I had a NoStorageConfigured_SecretsDisabled test that in my branch [here](https://github.com/Azure/azure-functions-host/commit/1c35d0bda882324df17e2d16e3d7c348b7d54957) that you should retain - it ensures we're running the SWA scenario and secrets aren't enabled.

Reply:
Right, this is why 10 AM brain should check what 1 AM brain did. That was an oversight; I'll add those back in.